modules/integration/tests-integration/tests/src/test/resources/testng.xml

```diff
@@ -49,6 +49,12 @@
         </classes>
     </test>
 
+    <test name="carbon-app-tests" preserve-order="true" parallel="false">
```

Comment:
The test name already exists. You need to use a unique test name for this testng.xml